it/helpers/ComponentSpecBase.scala

```diff
@@ -88,7 +89,7 @@ trait ComponentSpecBase extends TestSuite with CustomMatchers
 
   override def beforeEach(): Unit = {
     super.beforeEach()
-    wireMockServer.resetMappings()
+    WireMock.reset()
```

Comment:
Why this change in a test base class? What's the issue with instance of `wireMockServer`? Why using "default" `WireMock` management?

Reply:
wireMockServer.resetMappings() failed to properly reset systems in between tests causing false positives and false negatives through-out the tests

Reply:
Thanks, okay. Think about creating a tech debt to investigate and fix it while this looks-like-workaround solution is in place and does its job.